variant/schemas/ga4gh_vod.py

```diff
@@ -0,0 +1,270 @@
+"""Module for GA4GH Value Object Descriptor."""
+from pydantic import BaseModel
+from typing import List, Optional, Dict, Any, Type, Union
+from enum import Enum
+
+
+class Extension(BaseModel):
+    """Extend descriptions with other attributes unique to a content provider. -GA4GH"""  # noqa: E501
+
+    type = 'Extension'
+    name: str
+    value: List[Union[str, dict]]
+
+    @staticmethod
+    def schema_extra(schema: Dict[str, Any],
+                     model: Type['Extension']) -> None:
+        """Configure OpenAPI schema."""
+        if 'title' in schema.keys():
+            schema.pop('title', None)
+        for prop in schema.get('properties', {}).values():
+            prop.pop('title', None)
+        schema['example'] = {
+            'type': 'Extension',
+            'name': 'strand',
+            'value': ['+']
+        }
+
+
+class ValueObjectDescriptor(BaseModel):
+    """GA4GH Value Object Descriptor."""
+
+    id: str
+    type: str
+    label: Optional[str]
+    description: Optional[str]
+    value_id: Optional[str]
+    value: Optional[dict]
+    xrefs: Optional[List[str]]
+    alternate_labels: Optional[List[str]]
+    extensions: Optional[List[Extension]]
+
+    class Config:
+        """Configure model."""
+
+        orm_mode = True
```

Comment:
do these need ORM mode? (genuine question, I don't remember why we had to turn this on)

Reply:
Removed!